### ISSUE TITLE
Strange value for Asuswrt Devices Connected 

### ISSUE THREAD

**Issue by erkr** [ollo69/ha_asuswrt_custom#4]
Hi,

When I use AsusWrt with one of my access points (RTN-66U's, running a merlin LTS fork 3.0.0.4.374.43_33E7j9527), the number of devices as reported by the sensor Asuswrt Devices Connected doesn't make any sense.
It reports 14, while there is just one device connected to the local WiFi:
```
SSID: Home-xxxx
RSSI: NA    	  SNR: NA    	Noise: -90 dBm	Channel: 11
BSSID: 54:A0:50:DA:E7:F0	Capability: ESS ShortSlot 
HT Capable
	Extension channel: none
	Chanspec: 2.4GHz Channel Width 20MHz (0x100b)
Mode:   AP Only

Stations  (flags: P=Powersave Mode, S=Short GI, T=STBC, A=Associated, U=Auth, G=Guest)
------------------------------------------------------------------------------------------
MAC               IP Address        RSSI    Rx/Tx Rate   Connected   Flags
00:04:20:EB:C0:AA                  -56dBm     ??/72 Mbps  298:31:12   AU


SSID: Home-xxxx
RSSI: NA    	  SNR: NA    	Noise: -84 dBm	Channel: 100l
BSSID: 54:A0:50:DA:E7:F4	Capability: ESS 
HT Capable
	Extension channel: 102
	Chanspec: 5GHz Channel Width 40MHz (0xd866)
Mode:   AP Only

Stations  (flags: P=Powersave Mode, S=Short GI, T=STBC, A=Associated, U=Auth, G=Guest)
------------------------------------------------------------------------------------------
MAC               IP Address        RSSI    Rx/Tx Rate   Connected   Flags
```
And ca 85 clients are seen on the network of the WAN port (all connected via other AP's):
![image](https://user-images.githubusercontent.com/4363599/195571232-7526974d-5d91-40a1-b46d-51a9fb85695e.png)
That number 85 is not the actual number of connected devices. According my fritz box router, ca 25 devices are currently connected. 

I guess, in AP mode, only the devices on the local WiFi should count ?! The reported number of 14 is totally fuzzy for me.

Best Eric

**Comment by ollo69** [comment]
This is a very old firmware version. Maintain library for all firmware including very old as this it is quite impossible. Is not possible for you to upgrade to more recent version from [here](https://sourceforge.net/projects/asuswrt-merlin/files/RT-N66U/Release/)?

**Comment by erkr** [comment]
I will give it a go this weekend if you believe that helps 👍

**Comment by erkr** [comment]
@ollo69 
I checked the link you provided. These Merlin 380 based versions are no option. Reason is that the 374 based versions are the last version where ASUS allows free power settings for WIFI via the CLI interface. That is why I'm on a LTS fork. Updates in that LTS fork probably won't functionally extend what I currently have. Only bug and security fixes are provided.

**Comment by ollo69** [comment]
Understand your point, but as I said before I cannot maintain communication library compatibility for any firmware version.

**Comment by erkr** [comment]
Understood as well, but please note the 374 based versions are used by many owners, because of the power control limitations in the consecutive versions

**Comment by ollo69** [comment]
Do you have a link where I can download 374 version for AC68U?

**Comment by erkr** [comment]
You find the downloads for the supported routers over here:
https://www.snbforums.com/threads/fork-asuswrt-merlin-374-lts-overview.67510/
That includes the AC68U 👍

**Comment by ollo69** [comment]
Hope that my version match with one of the supported version, is the oldest Asus Router that I have. I will make some test as soon as I will have some available time, I will report in this thread my progress.

**Comment by erkr** [comment]
@ollo69 
If your router currently is on a newer 38x version, the asus restoration tool is required to downgrade back to 374 based versions

**Comment by ollo69** [comment]
I installed 374 version on my old AC68U. Unfortunately in this firmware HTTP API are not present, so is only possible to use SSH / Telnet protocol implemented by [aioasuswrt](https://github.com/kennedyshead/aioasuswrt).

**Comment by erkr** [comment]
Thanks for trying the 374 version!!! 
I don't mind using SSH / telnet. But is that related to the invalid device count? 
While playing with multiple AP's configured, I noted that the device trackers listed per AP don't reflect the devices that are connected via WiFi. It's some random subset of most wired devices on the LAN.
My intention was to use the device tracker of the AP's to do some device localisation. I guess that it's not going to fly.
Best Eric

**Comment by ollo69** [comment]
Yes, but this will require to work on library [aioasuswrt](https://github.com/kennedyshead/aioasuswrt), and in last period code owner of that library is not really active. For this reason I check if would be possible to manage this with http protocol that use different library managed by me. You could try to open an issue on [aioasuswrt](https://github.com/kennedyshead/aioasuswrt) and see if somebody will take this in charge.

**Comment by erkr** [comment]
Yes I will do that 👍

Update: https://github.com/kennedyshead/aioasuswrt/issues/87

**Comment by erkr** [comment]
That library project seems kind of dead indeed :-(
